packages/main/src/RatingIndicator.hbs

```diff
@@ -3,7 +3,7 @@
 	aria-roledescription="{{_ariaRoleDescription}}"
 	aria-valuemin="0"
 	aria-valuenow="{{value}}"
-	aria-valuemax="{{maxValue}}"
+	aria-valuemax="{{max}}"
```

Comment:
Why rename this when you changed the max to max-value in the slider / rangeSlider web components?